build.rs

```diff
@@ -450,27 +493,28 @@ fn configure(interpreter_version: &PythonVersion, lines: Vec<String>) -> Result<
 
     let mut flags = String::new();
 
+    if interpreter_version.implementation == PythonInterpreterKind::PyPy {
+        println!("cargo:rustc-cfg=PyPy");
+        flags += format!("CFG_PyPy").as_ref();
+    };
+
     if let PythonVersion {
         major: 3,
         minor: some_minor,
+        implementation: implementation,
```

Comment:
Looks like there's a warning about this being unused? https://travis-ci.org/PyO3/pyo3/jobs/511140863#L649

I'm seeing a lot of other warnings in that build as well, not sure how many of them are new with this PR.

Reply:
it's some `if let` patterns where I've simply added this is (but isn't actually needed inside). I'll make a round and clean up these warnings :)

Reply:
there are still a few imports (CStr), that rust complains that are unused, but if deleted cause a compile error (about missing CStr). This is strage (maybe it's my nightly version?)